### DISCUSSION TITLE
Sunbird Lern Building block 5.0.1 CSP sign-off

### DISCUSSION THREAD

**Discussion by Ak0094** [Sunbird-Lern/Community#49] [Documentation]
Hello All,

Please find the Sunbird lern Building block 5.0.1 CSP sign-off here.

Environment: [Dev lern](https://dev.lern.sunbird.org/)
Testing Status: Completed
Defects Dashboard: [Link](https://project-sunbird.atlassian.net/issues/?filter=-2&jql=issuetype%20%3D%20Defect%20AND%20Sprint%20%3D%20316%20order%20by%20created%20DESC)

Execution Sheet: [Link](https://docs.google.com/spreadsheets/d/12V_wAkVZFujRvS-VSDDfYqrOO6lm9HzOIe6Fd2biRmI/edit#gid=0)
Note: NA test cases in the execution sheet are related to a new course, certificate, and book TOC which cannot be tested in the dev lern due to environmental limitations.

Scope : [Link](https://project-sunbird.atlassian.net/issues/?filter=12695)
Building block sign-off template: [Link](https://docs.google.com/spreadsheets/d/1powm8Lh9oPLYzD_yLC5rjRyMZTS2TV5s7XnMvjFbfsE/edit#gid=0)

cc: @rhwarrier @reshmi-nair @chitranshu-keshav @ShailandraOjha @anunagaraj @Naveenkumar739

**Comment by rhwarrier** [comment]
[Ak0094](https://github.com/Ak0094) Thanks - assuming the test cases that cannot be tried out in Lern dev will be taken up in SB Ed staging, and will be executed end to end? We can't have a feature going out unless it has been signed off at some stage. 
cc : @chitranshu-keshav @reshmi-nair

**Comment by anunagaraj** [comment]
@reshmi-nair Please confirm if the sign-off includes the migration scripts and is handed over to the DevOps team.
[@Santhosh Gandham](mailto:santhosh.g@optit.in) [@Keshav Prasad](mailto:keshav.prasad@tarento.com) @G33tha Kindly confirm if you have received the migration scripts to run in the Sunbird ed environment.

CC @rhwarrier @Krishnaj20 @sharathkashyap @chitranshu-keshav @ashokreddy1208 @ShailandraOjha

**Comment by anunagaraj** [comment]
@reshmi-nair Please share an update on the above comment as we will not be able to start the regression until we have the confirmation on the migration scripts handed over to DevOps to run in the Sunbird ed environment.

CC @rhwarrier @Krishnaj20 @sharathkashyap @chitranshu-keshav @ashokreddy1208 @ShailandraOjha [@Santhosh Gandham](mailto:santhosh.g@optit.in) [@Keshav Prasad](mailto:keshav.prasad@tarento.com) @G33tha

**Reply by reshmi-nair** [reply]
@anunagaraj [Release notes of 5.0.1 ](https://lern.sunbird.org/use/release-notes/release-v-5.0.1)has the migration scripts also. It is handed over to DevOps team.
cc: @rhwarrier @G33tha [@Keshav Prasad](mailto:keshav.prasad@tarento.com)

**Reply by anunagaraj** [reply]
@G33tha [@Keshav Prasad](mailto:keshav.prasad@tarento.com) Kindly confirm if you have received the migration scripts to run in the Sunbird ed environment.

CC @rhwarrier @Krishnaj20 @sharathkashyap @chitranshu-keshav @ashokreddy1208 @ShailandraOjha [@Santhosh Gandham](mailto:santhosh.g@optit.in) [@Keshav Prasad](mailto:keshav.prasad@tarento.com) @G33tha

**Reply by G33tha** [reply]
@anunagaraj we are facing some issues during deployment, Learn team is looking into deployment issues.

**Reply by reshmi-nair** [reply]
@anunagaraj Deployment issues are resolved
